lib/jnpr/junos/utils/sw.py

```diff
@@ -241,7 +243,9 @@ def pkgaddISSU(self, remote_package, **kvargs):
 
     def _parse_pkgadd_response(self, rsp):
         got = rsp.getparent()
-        rc = int(got.findtext('package-result').strip())
+        # If <package-result> is not present, then assume success.
+        # That is, assume <package-result>0</package-result>
```

Comment:
@stacywsmith Please add a log for users reference when <package-result> is missing